lib/LaTeXML/Package/LaTeX.pool.ltxml

```diff
@@ -98,7 +98,7 @@ DefConstructor('\documentstyle OptionalSemiverbatim SkipSpaces Semiverbatim []',
     else {
       InputDefinitions('OmniBus', type => 'cls', noerror => 1,
         handleoptions => 1, options => $options,
-        after => Tokens(T_CS('\compat@loadpackages')));
+        after         => Tokens(T_CS('\compat@loadpackages')));
```

Comment:
Since I am seeing a bunch of whitespace linting diffs, here is the latest perltidy I'm running:

```
$ perltidy -v
This is perltidy, v20210402
```